xbmc/filesystem/FileCache.h

```diff
@@ -59,8 +59,7 @@ namespace XFILE
 
     IFile *GetFileImp();
 
-    std::string GetContent() override;
-    std::string GetContentCharset(void) override;
+    virtual const std::string GetProperty(XFILE::FileProperty type, const std::string &name = StringUtils::Empty) const override;
```

Comment:
No need for `virtual` here. No need for `StringUtils::Empty`. See my other comments.

Reply:
The removal of virtual is a matter of how peaple read foreign code. My eyes for example scan normally the beginning of member definitions, It could help some future devs scanning like me to get things faster together.

But, I don't want to spend much more time on this PR, so its. changed (PR will come soon)

Reply:
Sure, it's personal style. That's why I phrased this changes in almost all other places I wrote about this using "I would". ;-)